learner/tutorials/eclipse.md

```diff
@@ -0,0 +1,129 @@
+# Create a Open Eval client project using Eclipse
```

Comment:
Do you think you can link to this readme from outside? Like from `learner/README.md` maybe?